pom.xml

```diff
@@ -21,47 +21,31 @@
     </build>
 
     <dependencies>
-
-
         <!-- https://mvnrepository.com/artifact/com.microsoft.azure/adal4j -->
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>adal4j</artifactId>
-            <version>1.5.0</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-mgmt-keyvault</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
-        </dependency>
-        <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
+            <version>1.6.6</version>
         </dependency>
         <dependency>
-            <groupId>com.microsoft.azure</groupId>
-            <artifactId>azure-mgmt-storage</artifactId>
-            <version>1.3.0</version>
+            <groupId>com.azure</groupId>

```

Comment:
@alzimmermsft should we just change these samples to reference the BOM?

Reply:
Yes, this would be a good case where `azure-sdk-bom` should be used to manage dependencies, as long as not always using the latest versions is okay in samples.